forte/mrdsrg-spin-integrated/active_dsrgpt2.h

```diff
@@ -83,6 +83,9 @@ class ACTIVE_DSRGPT2 {
     /// Compute energy
     double compute_energy();
 
+    /// Compute gradient
+    psi::SharedMatrix compute_gradient() override;
+
```

Comment:
```suggestion
```

Reply:
You may notice that such definition appears in multiple dsrg files. Actually you suggested me to do so 1 year ago, and the fact is that the code doesn't work anymore if these codes are removed. There will be some errors like :ImportError: dlopen(/Users/shuhe.wang/src/forte/forte/forte.cpython-36m-darwin.so, 2): Symbol not found: __ZN5forte10DSRG_MRPT316compute_gradientEv
  Referenced from: /Users/shuhe.wang/src/forte/forte/forte.cpython-36m-darwin.so
  Expected in: flat namespace

Reply:
I am surprised because this class is not even part of `MASTER_DSRG` and why do you actually need `compute_gradient()`? Please be careful with the resolve button because the suggestion is clearly not taken.